src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -843,6 +967,41 @@ ApiGlobe.prototype.callbackLayerChanged = function callbackLayerChanged() {
     this.dispatchEvent(eventLayerChanged);
 };
 
+ApiGlobe.prototype.addEventListenerCenterChanged = function addEventListenerCenterChanged() {
+    this.getCenterStatus('new');
```

Comment:
No need here, it's just a initialization so the old status will be good in the function `callbackCenterChanged`

It's the same for the events `zoomchanged` and `orientationchanged`